keyboards/handwired/sick_pad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,65 @@
+#include "kb.h"
```

Comment:
```suggestion
#include QMK_KEYBOARD_H
```

Prefer the more generic `#include QMK_KEYBOARD_H` here.